manifests/noderesourcetopology/crd.yaml

```diff
@@ -3,7 +3,6 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    api-approved.kubernetes.io: https://github.com/kubernetes/enhancements/pull/1870 # edited manually
```

Comment:
I think this annotation has to stay to be able to create CRD in `k8s.io` domain

Reply:
I agree. But it should be added automatically (not manually) if possible, hence: https://github.com/k8stopologyawareschedwg/noderesourcetopology-api/pull/29

Reply:
Ok, I can see you got this covered 😄

Reply:
hopefully! :)